ElectronicObserver/Data/Constants.cs

```diff
@@ -1001,20 +991,12 @@ public static string GetExpeditionResult(int value)
 	#endregion
 
 	#region Servers
-	public class KCServer
+	public class KCServer(int num, string name, string jp, string ip)
```

Comment:
Probably, but that one is actually unrelated to drop record viewer, I only made the change cause sonar was complaining because I updated editorconfig.
So I wouldn't do that refactoring in this PR.

Reply:
okay